main/res/values/strings.xml

```diff
@@ -1660,4 +1660,9 @@
     <string name="ask_again">Ask me again</string>
     <string name="location_permission_request_explanation">c:geo needs access to the GPS on your device to locate your position and calculate distance and direction to geocaches. It cannot be used without this permission.</string>
     <string name="storage_permission_request_explanation">c:geo will write data onto your phone storage or SD card as soon as you save geocaches for offline use. Furthermore c:geo will use your phone storage for import and export of files and reading of offline maps. It cannot be used without this permission.</string>
+
+    <!-- colors -->
```

Comment:
AFAIK such single words should also start with a capital letter in english if they are used as a heading or selection list...but I did not look it up.

Reply:
I'm unsure, and found different opinions on the web on this. Some examples:
- Start every sentence with a capital letter? Obvious - but does a single word form a sentence? (eg: https://www.thoughtco.com/guidelines-for-using-capital-letters-1691724)
- Capitalize the first and the last word (eg: https://capitalizemytitle.com/ - New York Times style and Wikipedia style)
- List entities start with _lowercase_ letters if the don't consist of a whole sentence (eg: https://getitwriteonline.com/articles/vertical-lists/).

???

Reply:
Yes, its difficult. Looking at our app we IMHO tend to capitalize menu entries independently if its a whole sentence or a fragment. Therefore we should just continue this style. And additionally lower case first letter hurts my german language focussed brain ;)

Reply:
ok, capitalization won :-)